.travis.yml

```diff
@@ -1,16 +1,15 @@
 language: node_js
 env:
   - FORCE_COLOR=true
-cache: npm
 notifications:
   email: false
 node_js:
-  - 10.14
+  - 10.18
   - 12
   - node
-install: npm install
+install: yarn install
 script:
-  - npm run validate
+  - yarn validate
```

Comment:
Why the change to yarn? Is it needed?

Reply:
For some reason there were problems with `npm install`. It looked like it still tried to install `@types/testing-library__jest-dom` which were wrong at some places which failed tests. I don't know why it installed them (I'm not sure why they were included since they are not a dependency anymore -- it might be travis cache/and/or lack of lockfile)

Reply:
Made some changes so yarn is no longed needed.